script/Ciel/Day 13/13_01_01_CIEL13_5.txt

```diff
@@ -777,16 +777,18 @@ Depravity upon our prisoner.
 -- 遊覧腐爛のツギハギモザイク、
 // This doesn't feel quite right because I'd expect it to be more like a 'sightseeing rot' or something but what the hell would that even mean
 // Wild guess but is this 遊覧 supposed to be interpreted as pleasure somehow? As in 遊覧船 -> pleasure cruise? Kinda grasping at straws.
+// Putrefying artwork in mosaic patchwork.
 %{i}
-Putrefying panoramas in patchwork mosaic.
+Putrefying prosaic in patchwork mosaic.
 %{/i}
 }
 [sha:edb1f81fdd9883e3a823e9f96f06b2fc94f822f4]{
 -- Page 562, Offset 34889.
 -- 一夜ばかりの幻も　終わらぬものなら不治の<恋|どく>。
-%{i}Incurable %{/i}
-<%{i}poison%{/i}|%{i}l o v e   %{/i}>
-%{i} be, the never-ending night of fantasy.%{/i}
+%{i}
+The poison of love has no mending,
+were the night of fantasy never-ending.
```

Comment:
Think the original might convey the actual idea better. Or maybe reordering works?
> Should the night of fantasy never end,
> then never shall love's poison mend.

Reply:
You might need to add a space before the second line

Reply:
I like Ross' suggestion, but what do you mean Volvo?

Reply:
I think just pointing out that the original would have rendered like this `mending,were` since the second line has no leading space. Looks like your update fixed it.